sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala

```diff
@@ -268,8 +262,7 @@ object PROCESS_TABLES extends QueryTest with SQLTestUtils {
         .filter(_.contains("""<a href="spark-"""))
         .filterNot(_.contains("preview"))
         .map("""<a href="spark-(\d.\d.\d)/">""".r.findFirstMatchIn(_).get.group(1))
-        .filter(_ < org.apache.spark.SPARK_VERSION)
-        .filterNot(skipReleaseVersions.contains).toImmutableArraySeq
```

Comment:
Maybe we can keep this function. I am not sure if there will be similar needs in the future that allow us to reuse it again.

Reply:
Okay

Reply:
Updated.

Reply:
We should update pr title and pr description to match the change

Reply:
Yea, updated.